packs/agents-of-edgewatch-bestiary/grabble-forden.json

```diff
@@ -3113,11 +3113,11 @@
                         "toggleable": true
                     },
                     {
-                        "itemType": "spell",
                         "key": "DamageAlteration",
                         "mode": "override",
                         "predicate": [
                             "poison-conversion",
+                            "item:type:spell",
```

Comment:
Figured I needed to tighten this up since it doesn't predicate by slug, and we don't want to trigger on "inline-damage" that isn't a spell.